### DISCUSSION TITLE
Search feature

### DISCUSSION THREAD

**Discussion by Osiris9999** [Byron/google-apis-rs#289] [Q&A] [answered]
Can I use this API to search youtube and get results? I saw that the search is not a hyperlink in the docs.

**Comment by Byron** [comment, marked as answer]
Yes, despite the link missing for some reason it's possible to [search](https://docs.rs/google-youtube3/2.0.4+20210330/google_youtube3/api/struct.SearchMethods.html), both using the API as well as the docs by hitting the `S` key, the `?` key or using the search field directly.

<img width="1070" alt="Screenshot 2021-06-22 at 07 49 48" src="https://user-images.githubusercontent.com/63622/122841725-6ddba680-d32e-11eb-976a-7e35b21accee.png">
